website/docs/introduction/java/about-workshop.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Java Workshop Overview
+sidebar_position: 1
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+In this workshop, learn how to build cloud-native Java applications and deploy to Amazon Elastic Kubernetes Service (EKS) with best practices and performance-optimization techniques. Get hands-on experience using Amazon EKS, Amazon Corretto, Amazon ECR, GraalVM, Spring Boot, and more.
+
+The workshop covers a variety of AWS services and tools and provides an introduction to Java related concepts. Below you can find a broad overview of services that are covered throughout the modules:
+
+![java-on-aws-eks](./images/java-on-amazon-eks.png)
```

Comment:
Just a comment that can be fixed later on.
This image shows incoming requests received by ALB and sent to aws-lb-controller. Actually, aws-lb-controller is not in the network flow, ALB sends requests directly to App pod/container (or K8s svc).
Users are often confused thinking that network traffic goes through the aws-lb-controller, which is not correct.